examples/test.c

```diff
@@ -0,0 +1,4 @@
+int mytest()
+{
+	return TEST;
```

Comment:
What happens if TEST is undefined? Maybe add a fallback #ifndef and call this `TEST_CFFI_CFLAG` or something.